### DISCUSSION TITLE
List Of Games With Abnormal PCRTC Settings

### DISCUSSION THREAD

**Discussion by tadanokojin** [PCSX2/pcsx2#4107] [Information]
I want a list of games which have strange PCRTC settings (double height, etc).
Basically anything other than output single display

- Silent Hill 2 (double height)
- Pool Paradise (double height)
- DMC (output subtitles on separate display with offset)
- Makai Kingdom (half height/progressive)
- The Suffering series (feedback write/ offset) - [hack PR/info](https://github.com/PCSX2/pcsx2/pull/2991)

Others (need more info/testing)
- KOF 2000 (improper height)
- Metal Slug 3 (improper height) (NTSC 60hz only)
- Twisted Metal FMV (improper width)
- ICO FMV (improper width)

**Comment by ghost** [comment]
Ratchet and Clank up your arsenal / Deadlocked main menu suffer of a improper height in the PAL releases.

**Comment by ghost** [comment]
Arctic Thunder (US) set double height for Videos (640x896).

**Comment by felixthecat1970** [comment]
come from alone in the dark issue, i hope this info mean something:

Tested 1.7 dev 1201 / default-safe mode and debug settings

Observations:
- this games has more height screen hide 512x448>512x512 or 640x448>640x480 (meanwhile i made .pnachs files are available in forums)
Silent Hill 2 (USA) (En,Ja,Fr,De,Es,It) (v2.01) *this version height looks ok 
Silent Hill 3 (USA) (En,Ja,Fr,De,Es,It,Ko)
Rumble Fish, The (Japan)
Metal Slug 4 (USA)

- Ico (USA) *this version videos look ok

- Games with weird (half height/progressive) - weird values resolution
Sega Ages 2500 Series Vol. 27 - Panzer Dragoon (Japan) 
Gunbird Special Edition (Europe) (En,Fr,De)
Chain Dive (Japan) *videos 640x384?
*NeoGeo Online Series games change to 320x224 or 640x448 except this title:
 NeoGeo Online Collection Vol. 2 - Bakumatsu Roman - Gekka no Kenshi 1, 2 (Japan) (En,Ja,Es,Pt) 320x448?
Tekken 5 (Europe, Australia) (En,Fr,De,Es,It) - arcade games 1/2 512x256 (game clear memory each game is played more elfs?)
Alone in the Dark - The New Nightmare (Europe) (En,Fr,De,Es,It) - (already posted issue)

- Games with 640x478 2pixel off  looks like is a antialias mode, 
in game options you turn it off and game display 640x480

Hokuto no Ken Shinpan no Sousousei Kengou Retsuden (Japan)
Sengoku Basara X (Japan)
